GNUmakefile

```diff
@@ -20,13 +20,13 @@ GOFILES_NOVENDOR = $(shell find . -type f -name '*.go' -not -path "./vendor/*")
 # Specify the name for the binaries
 MAYACTL=mayactl
 APISERVER=maya-apiserver
-AGENT=maya-agent
+POOL_MGMT=cstor-pool-mgmt
```

Comment:
Can we use openebs/community/feature-demos/ instead of openebs/openebs/k8s/crd-samples ??

Reply:
Do you mean to move the yaml files to a new directory - openebs/community/feature-demos/ ?